assets/css/about.css

```diff
@@ -0,0 +1,359 @@
+/* Import the specific font family we are going to use */
+@import url("http://fonts.cdnfonts.com/css/cocogoose?styles=44178");
+
+*,
+*::before,
+*::after {
+  font-family: COCOGOOSE, sans-serif;
+  font-weight: lighter;
+  font-size: small;
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+html,
+body {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  overflow-x: hidden;
+}
+
+ul,
+li,
+a {
+  padding-left: 0;
+  list-style: none;
+  text-decoration: none;
+  color: inherit;
+}
+
+.menu-icon {
+  width: 4em;
+  margin: 1em 2em;
```

Comment:
Kindly the hamburger should be on the `right` instead of the left. and it should disappear when the user go down on another section. i understand that you had put its `position fixed` that's why it doesn't disappear even if you go down in a section. please fix that

Reply:
Hi @vic778, from the design template.  It is not on the right it's on the left.!!!!!
https://www.behance.net/gallery/29845175/CC-Global-Summit-2015

Reply:
I removed the position fixed now

Reply:
what if the next reviewer tells me to re-put it to left as the original design ??? I have the last review attempt